ipn/store.go

```diff
@@ -76,6 +76,8 @@ func ServeConfigKey(profileID string) StateKey {
 type ServeConfig struct {
 	// TCP are the list of TCP port numbers that tailscaled should handle for
 	// the Tailscale IP addresses. (not subnet routers, etc)
+	//
+	// TODO(shayne,bradfitz): can this be uint16?
```

Comment:
yeah, probably.

pushes some conversions/checks elsewhere instead, but probably not too many